pyeudiw/tests/storage/test_mongo_storage.py

```diff
@@ -19,7 +19,8 @@ def test_mongo_connection(self):
 
         assert self.storage.db is not None
         assert self.storage.client
-        assert self.storage.collection is not None
+        assert self.storage.sessions is not None
+        assert self.storage.chains is not None
```

Comment:
please change chains with attestations

Reply:
Resolved in [922d2a2](https://github.com/italia/eudi-wallet-it-python/pull/94/commits/922d2a25085f4693c38f37c40d90db879183465b)